### PR TITLE
Added test for multiple sort capability for filings

### PR DESCRIPTION
**Pull request by jontours**
Ready for review.

### CODE REVIEW

**Comment by dogweather** [comment]
I see issues with the test — let me a commit or two to see what you think, instead of writing it out in English -

File: tests/test_filings.py [hunk 1]
```diff
@@ -108,6 +108,21 @@ def test_sort_bad_column(self):
...
+    def test_sort_multi_column(self):
+        [
+            factories.FilingsFactory(beginning_image_number=2, coverage_end_date='2017-01-01'),
+            factories.FilingsFactory(beginning_image_number=1, coverage_end_date='2017-01-01'),
+            factories.FilingsFactory(beginning_image_number=0, coverage_end_date='2017-01-02'),
+        ]
+
+        results = self._results(api.url_for(FilingsList, sort=['coverage_end_date', 'beginning_image_number']))
+        self.assertEqual(results[0]['beginning_image_number'], '1')
+        self.assertEqual(results[2]['beginning_image_number'], '0')
```
Comment (dogweather) [inline tests/test_filings.py]: Ok, actually, I've got a couple of questions before I can contribute. (And this points to the clarity I think we can add with some edits.) 

What are these asserts testing for? Is it that when sorting by multiple fields, the secondary field is correctly ordered?

File: tests/test_filings.py [hunk 2]
```diff
@@ -108,6 +108,21 @@ def test_sort_bad_column(self):
...
+        results = self._results(api.url_for(FilingsList, sort=['coverage_end_date', '-beginning_image_number']))
+        self.assertEqual(results[0]['beginning_image_number'], '2')
```
Comment (dogweather) [inline tests/test_filings.py]: This one is testing for reverse sort order?

File: tests/test_filings.py [hunk 3]
```diff
@@ -108,6 +108,21 @@ def test_sort_bad_column(self):
...
+        results = self._results(api.url_for(FilingsList, sort=['-coverage_end_date', '-beginning_image_number']))
+        self.assertEqual(results[0]['beginning_image_number'], '0')
```
Comment (dogweather) [inline tests/test_filings.py]: And this final one ... I can't actually tell what this is testing for 🦊

File: tests/test_filings.py (hunk 1 above)
Comment (jontours) [inline tests/test_filings.py]: Yes, that is indeed what this assert is for.

File: tests/test_filings.py (hunk 3 above)
Comment (jontours) [inline tests/test_filings.py]: This test is that good old, regular sort still trumps the secondary sort.

Comment (dogweather) [inline tests/test_filings.py]: Ok, thanks. I'm on vacation now, so won't be submitting a commit. But my suggestion is to break this up into three tests with descriptive names containing what you said. E.g., 

`def test_old_regular_sort_still_trumps_secondary_sort`

I.e., test names which document the expected outcome.

Comment (jontours) [inline tests/test_filings.py]: Sure thing!

**Comment by ccostino** [comment]
Awesome work, thanks @jontours and @dogweather!

**Pull request merged**